framework/src/main/java/fr/pilato/elasticsearch/crawler/fs/framework/bulk/FsCrawlerBulkRequest.java

```diff
@@ -24,17 +24,17 @@
 
 public abstract class FsCrawlerBulkRequest<T extends FsCrawlerOperation<T>> {
 
-    private final List<T> operations = new ArrayList<>();
```

Comment:
I don't think we should change that.

Reply:
In fr.pilato.elasticsearch.crawler.fs.framework.bulk.FsCrawlerBulkProcessor.java, we were not able to access "getJson" method written in "ElasticsearchIndexOperation" class in "add" method written in FsCrawlerBulkProcessor.java. To achieve this, we changed data type of argument passed to "Object" and later on casted the object for respective class to access it's methods. If you can help in achieving the same in any other way without modifying data type, then it will be helpful